diff --git a/node_modules/react-native-pell-rich-editor/src/RichEditor.js b/node_modules/react-native-pell-rich-editor/src/RichEditor.js
index 74de536..8471610 100755
--- a/node_modules/react-native-pell-rich-editor/src/RichEditor.js
+++ b/node_modules/react-native-pell-rich-editor/src/RichEditor.js
@@ -262,8 +262,10 @@ export default class RichTextEditor extends Component {
         return (
             <>
                 <WebView
+                    textZoom={100}
                     useWebKit={true}
                     scrollEnabled={false}
+                    androidHardwareAccelerationDisabled={true}
                     hideKeyboardAccessoryView={true}
                     keyboardDisplayRequiresUserAction={false}
                     nestedScrollEnabled={!useContainer}
@@ -473,6 +475,6 @@ const styles = StyleSheet.create({
     },
 
     webview: {
-        backgroundColor: "transparent"
+        backgroundColor: "transparent",
     }
 });
diff --git a/node_modules/react-native-pell-rich-editor/src/RichToolbar.js b/node_modules/react-native-pell-rich-editor/src/RichToolbar.js
index 6882aa8..ce8133c 100755
--- a/node_modules/react-native-pell-rich-editor/src/RichToolbar.js
+++ b/node_modules/react-native-pell-rich-editor/src/RichToolbar.js
@@ -197,7 +197,7 @@ export default class RichToolbar extends Component {
             case actions.setHR:
             case actions.indent:
             case actions.outdent:
-                editor.showAndroidKeyboard();
+                // editor.showAndroidKeyboard();
                 editor.sendAction(action, 'result');
                 break;
             case actions.insertImage:
diff --git a/node_modules/react-native-pell-rich-editor/src/editor.js b/node_modules/react-native-pell-rich-editor/src/editor.js
index 4252bd9..848e578 100644
--- a/node_modules/react-native-pell-rich-editor/src/editor.js
+++ b/node_modules/react-native-pell-rich-editor/src/editor.js
@@ -34,7 +34,7 @@ function createHTML(options = {}) {
         inputListener = false,
         autoCapitalize = 'off',
         enterKeyHint = '',
-        autoCorrect = false,
+        autoCorrect = true,
         defaultParagraphSeparator = 'div',
         // When first gaining focus, the cursor moves to the end of the text
         firstFocusEnd = true,
@@ -46,7 +46,7 @@ function createHTML(options = {}) {
 <html>
 <head>
     <title>RN Rich Text Editor</title>
-    <meta name="viewport" content="user-scalable=1.0,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0">
+    <meta name="viewport" content="width=device-width,initial-scale=1.0,minimum-scale=1.0,maximum-scale=1.0">
     <style>
         ${initialCSSText}
         * {outline: 0px solid transparent;-webkit-tap-highlight-color: rgba(0,0,0,0);-webkit-touch-callout: none;box-sizing: border-box;}
@@ -435,7 +435,7 @@ function createHTML(options = {}) {
             content.id = 'content';
             content.contentEditable = true;
             content.spellcheck = false;
-            content.autofocus = true;
+            content.autofocus = false;
             content.enterKeyHint = '${enterKeyHint}';
             content.autocapitalize = '${autoCapitalize}';
             content.autocorrect = ${autoCorrect};
@@ -580,13 +580,16 @@ function createHTML(options = {}) {
                     exec("insertText", text);
                 }
             });
-            addEventListener(content, 'compositionstart', function(event){
-                compositionStatus = 1;
-            })
-            addEventListener(content, 'compositionend', function (event){
-                compositionStatus = 0;
-                paragraphStatus && formatParagraph(true);
-            })
+
+            // these make a mess on Android, and are not even used on iOS
+            // so it's better to disable them for now
+            // addEventListener(content, 'compositionstart', function(event){
+            //     compositionStatus = 1;
+            // })
+            // addEventListener(content, 'compositionend', function (event){
+            //     compositionStatus = 0;
+            //     paragraphStatus && formatParagraph(true);
+            // })
 
             var message = function (event){
                 var msgData = JSON.parse(event.data), action = Actions[msgData.type];
