diff --git a/node_modules/react-native-double-tap/index.js b/node_modules/react-native-double-tap/index.js
index 812a47e..4afacf2 100644
--- a/node_modules/react-native-double-tap/index.js
+++ b/node_modules/react-native-double-tap/index.js
@@ -1,5 +1,5 @@
 import React, { Component } from "react";
-import { TouchableOpacity } from "react-native";
+import { ScrollView } from "react-native";
 
 export default class DoubleClick extends Component {
   constructor(props) {
@@ -50,10 +50,11 @@ export default class DoubleClick extends Component {
   };
 
   render() {
+    const { reference, onScroll } = this.props;
     return (
-      <TouchableOpacity onPress={this._onPress}>
+      <ScrollView scrollEventThrottle={16} ref={reference} onScroll={onScroll} bounces contentContainerStyle={{ flexGrow: 1 }} showsVerticalScrollIndicator={false} onTouchEnd={this._onPress}>
         {this.props.children}
-      </TouchableOpacity>
+      </ScrollView>
     );
   }
 
